src/Bridge/Doctrine/Orm/Extension/EagerLoadingExtension.php

```diff
@@ -66,18 +82,20 @@ public function getMetadataProperties(string $resourceClass) : array
      *
      * @param QueryBuilder $queryBuilder
      * @param string       $resourceClass
+     * @param array        $context       the serialization context
```

Comment:
This is not the "serialization context", but `$options` passed to the factory. See [PropertyMetadataFactoryInterface](https://github.com/api-platform/core/blob/c1090d136aae14c384a3b6646de2f8e7cc110830/src/Metadata/Property/Factory/PropertyMetadataFactoryInterface.php#L35).